internal/gps/solve_basic_test.go

```diff
@@ -486,11 +486,11 @@ var basicFixtures = map[string]basicFixture{
 	},
 	"simple dependency tree": {
 		ds: []depspec{
-			mkDepspec("root 0.0.0", "a 1.0.0", "b 1.0.0"),
```

Comment:
omg this is awful and never even occurred to me i can't believe you did all this i hope you wrote a script you're amazing